modules/UI/shared_video/SharedVideoThumb.js

```diff
@@ -65,9 +65,9 @@ SharedVideoThumb.prototype.createContainer = function(spanId) {
 };
 
 /**
- * Sets the display name for the thumb.
+ * Updates the display name component for the given video span id.
```

Comment:
There is no given span id. When I write "for the given" it usually refers to parameters, I find this comment confusing.